### PR TITLE
PMM-5549 [FB] Grafana 6.7.3

### PR DESCRIPTION
**Pull request by Sevlin**
Updated grafana to 6.7.3
---
Related PRs:

- [x] [pmm-server-packaging](https://github.com/percona/pmm-server-packaging/pull/129)

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-819-1b6cdfb
client docker - perconalab/pmm-client-fb:PR-819-1b6cdfb
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-819-1b6cdfb.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-819-04294dd
client docker - perconalab/pmm-client-fb:PR-819-04294dd
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-819-04294dd.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-819-ce5a3be
client docker - perconalab/pmm-client-fb:PR-819-ce5a3be
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-819-ce5a3be.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-819-06212c9
client docker - perconalab/pmm-client-fb:PR-819-06212c9
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-819-06212c9.tar.gz

File: build/bin/prepare-node-modules [hunk 1]
```diff
@@ -36,18 +36,14 @@ update_grafana_node_modules() {
...
-        sudo /usr/bin/curl --silent --location https://rpm.nodesource.com/setup_10.x | sudo bash -
-        sudo yum clean all
-        sudo yum -y install nodejs
 
-        sudo npm --verbose install -g yarn
         rm -rf BUILD/grafana-*
         sudo chown builder:builder SOURCES/grafana-*
         rpmbuild --nodeps -bp SOURCES/grafana.spec || :
         pushd BUILD/grafana-*
+        sleep 5
```
Comment (AlekSi) [inline build/bin/prepare-node-modules:44]: ?

**Comment by AlekSi** [comment]
Please note that there are changes that should be merged (build scripts) and changes that should not (.gitmodules, submodules)

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-819-72cf2d3
client docker - perconalab/pmm-client-fb:PR-819-72cf2d3
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-819-72cf2d3.tar.gz

**Pull request merged**